iroh/examples/sync.rs

```diff
@@ -502,6 +546,17 @@ pub enum Cmd {
     WatchCancel,
     /// Show stats about the current session
     Stats,
+    /// Stress test with the hammer
+    Hammer {
```

Comment:
Now that we're at four arguments, I'd prefer to turn them into options. Then you can also add default values if needed.
`#[clap(short, long)]` etc